.travis.yml

```diff
@@ -1,19 +1,16 @@
 dist: trusty
 language: python
 python:
-- '2.6'
 - '2.7'
 - '3.5'
 - '3.6'
   # - 'pypy'
 install:
-- pip install requests==2.10.0 flake8==2.6.2 nose mock tox
-- if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install ordereddict unittest2;
-  fi
+- pip install -r requirements.txt
```

Comment:
can we rename this requirements_dev.txt or something that denotes it is for dev/testing?

Reply:
Yep